src/arena.rs

```diff
@@ -240,6 +250,52 @@ impl<T> Arena<T> {
             marker: PhantomData,
         }
     }
+
+    #[allow(unused_variables)]
+    pub fn get_span(&self, handle: Handle<T>) -> &Span {
+        #[cfg(feature = "span")]
+        {
+            return self.span_info.get(handle.index()).unwrap_or(&Span::Unknown);
+        }
+        &Span::Unknown
+    }
+
+    #[allow(unused_variables)]
+    pub fn set_span(&mut self, handle: Handle<T>, span: Span) {
+        #[cfg(feature = "span")]
+        {
+            self.span_info[handle.index()].clone_from(&span);
```

Comment:
nit: if this requires a reference anyway, let's have the argument as a reference

Reply:
Nuked from orbit.